README.md

```diff
@@ -1,6 +1,6 @@
 # JSON.jl
 
-This package provides for parsing and printing JSON in pure Julia.
+This package provides for parsing and printing JSON in pure Julia. See also alternative packages [JSON3.jl](https://github.com/quinnj/JSON3.jl), and [Serde.jl](https://github.com/bhftbootcamp/Serde.jl) (based on the Serde Rust library that handles more formats than just JSON).
```

Comment:
The claim that Serde.jl is based on the rust library is blatantly false and misleading. In fact, Serde.jl uses this package.